test/gtest/ucp/test_ucp_error_handling.cc

```diff
@@ -9,14 +9,112 @@
 
 class test_ucp_error_handling : public test_ucp_tag {
 public:
+    test_ucp_error_handling() : err_cntr(0ul), err_status(UCS_OK) {}
```

Comment:
need to make a test with 2 pairs, check that after 1 pair fails the other one still works (to expose the config bug above)